ui/tests/acceptance/enterprise-transform-test.js

```diff
@@ -108,7 +101,9 @@ module('Acceptance | Enterprise | Transform secrets', function (hooks) {
   });
 
   test('it can create a transformation and add itself to the role attached', async function (assert) {
-    const backend = await mount();
+    await visit('/vault/settings/mount-secret-backend');
```

Comment:
This `visit` can also be called by hitting the page object `mount-secret-backend`. I opted to not do this because it's continuing the use of a page objects. Further this PR attempts to keep scope to the `mount-backend-form` , though there is some close overlap between these two—visiting the route mount-secret-backend loads the mount-backend-form 🫠

Reply:
These three lines are repeated, however pulling it out into a function doesn't really make sense due to the order of operations. I first visit, then set the backend local var (need access within the test scope) and then mount.